solved_exercises/dani/02-tuples.py

```diff
@@ -0,0 +1,43 @@
+import itertools
+'''*********************************************************************************************'''
+print('*****************Check wether an element exists within a tuple******************')
+my_tuple = 1, 2, 3, 3, 4, 5, 6
+print('my_tuple is', my_tuple)
+a=3
+b=7
+print('Version1')
+print (a in my_tuple and (a," is in my_tuple") or (a," isn't in my_tuple"))
+print (b in my_tuple and (b," is in my_tuple") or (b," isn't in my_tuple"))
+'''---------------------------------------------------------------------------'''
+print('Version2')
+print(my_tuple.count(a) and (a," is in my_tuple") or (a," isn't in my_tuple"))
+print(my_tuple.count(b) and (b," is in my_tuple") or (b," isn't in my_tuple"))
+
```

Comment:
+1 for the creativity to get a second version, but -1 for the complicated printing 📟